<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QLabel" name="dateTimeLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255); 
font: 16pt &quot;Verdana&quot;;</string>
   </property>
   <property name="text">
    <string>Add Task</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>440</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton{
color: rgb(255, 255, 255); 
font: 16pt &quot;Verdana&quot;;
background-color: rgb(181, 181, 181);
border-radius: 10px;
}

QPushButton:hover {
color: rgb(255, 159, 102);
}</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255); 
font: 16pt &quot;Verdana&quot;;</string>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255); 
font: 16pt &quot;Verdana&quot;;</string>
   </property>
   <property name="text">
    <string>Machine ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dialogAddTaskButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>440</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton{
color: rgb(255, 255, 255); 
font: 16pt &quot;Verdana&quot;;
background-color: rgb(181, 181, 181);
border-radius: 10px;
}

QPushButton:hover {
color: rgb(255, 159, 102);
}</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>80</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255); 
font: 16pt &quot;Verdana&quot;;</string>
   </property>
   <property name="text">
    <string>Date of doing task</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="doneTaskCalendar">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="typeCombo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Verdana&quot;;
background-color: rgb(181, 181, 181);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="machineCombo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Verdana&quot;;
background-color: rgb(181, 181, 181);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255); 
font: 16pt &quot;Verdana&quot;;</string>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="descriptionField">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>221</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Verdana&quot;;
background-color: rgb(181, 181, 181);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255); 
font: 16pt &quot;Verdana&quot;;</string>
   </property>
   <property name="text">
    <string>Parts</string>
   </property>
  </widget>
  <widget class="QComboBox" name="partsCombo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Verdana&quot;;
background-color: rgb(181, 181, 181);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
